back/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM node:16-alpine AS builder
+
+WORKDIR /usr/src/app/back
+ENV PATH /node_modules/.bin:$PATH
+
+COPY package.json /
```

add package-lock too

```suggestion
COPY package.json /
COPY package-lock.json /
```